spec/controllers/users/sessions_controller_spec.rb

```diff
@@ -55,12 +55,23 @@
     context 'when user is present' do
       it 'sets live key to true' do
         sign_in_as_user
+        session[:session_expires_at] = Time.current + 10
         get :active
 
         json ||= JSON.parse(response.body)
 
         expect(json['live']).to eq true
       end
+
+      it 'respects session_expires_at' do
+        sign_in_as_user
+        session[:session_expires_at] = Time.current - 1
+        get :active
```

Comment:
`get :active` should never log the user out.

`get :timeout` should log the user out.